dist/images/install.sh

```diff
@@ -1921,15 +1921,13 @@ spec:
           readinessProbe:
             exec:
               command:
-                - bash
-                - /kube-ovn/kube-ovn-controller-healthcheck.sh
+                - ./kube-ovn-controller-healthcheck
```

Comment:
We should use the absolute path `/kube-ovn/kube-ovn-controller-healthcheck` instead.